internal/signer/linux/pkcs11/pkcs11_test.go

```diff
@@ -52,23 +51,6 @@ func TestParseHexStringFailure(t *testing.T) {
 	}
 }
 
-func TestEncryptRSA(t *testing.T) {
```

Comment:
What's the context for this? Is it a duplicate or something?

Reply:
This is something I learned from my PR on go-pkcs11 from Eric's comments. I originally had two variations of TestEncry, one encrypting with go-pkcs11 and one with the crypto library. Eric said that it was best practice to test the public API wherever possible, so I chose to consolidate the TestEncrypt into one function. Go-pkcs11 also no longer supports encryption so having two tests for the private APIs was not needed.